PlanPluginBridge/src/main/java/com/djrapitops/pluginbridge/plan/nucleus/NucleusData.java

```diff
@@ -0,0 +1,311 @@
+                        User noterUser = getUser(note.getNoter().get());
+                        if (noterUser != null) {
+                            noter = noterUser.getName();
+                        }
+                    }
+
+                    notesTable.addRow(noter, note.getNote());
+                }
+
+                inspectContainer.addTable("Notes", notesTable);
+            }
+        }
+
+        if (NucleusAPI.getWarningService().isPresent()) {
+            NucleusWarningService warningService = NucleusAPI.getWarningService().get();
+
+            List<Warning> warnings = warningService.getWarnings(user);
+            inspectContainer.addValue(getWithIcon("Warning count", Icon.called("flag").of(Color.AMBER)), warnings.size());
+
+            if (!warnings.isEmpty()) {
+                TableContainer warningsTable = new TableContainer(
+                        getWithIcon("Warner", Icon.called("exclamation").of(Family.SOLID)),
+                        getWithIcon("Reason", Icon.called("sticky-note").of(Family.SOLID))
+                );
+
+                warningsTable.setColor("amber");
+
+                for (Warning warning : warnings) {
+                    String warner = "Unknown";
+                    if (warning.getWarner().isPresent()) {
+                        User warnerUser = getUser(warning.getWarner().get());
+                        if (warnerUser != null) {
+                            warner = warnerUser.getName();
+                        }
+                    }
+
+                    warningsTable.addRow(warner, warning.getReason());
+                }
+
+                inspectContainer.addTable("Warnings", warningsTable);
+            }
+        }
+
+        return inspectContainer;
+    }
+
+    @Override
+    public AnalysisContainer getServerData(Collection<UUID> uuids, AnalysisContainer analysisContainer) {
+        if (NucleusAPI.getWarpService().isPresent()) {
+            NucleusWarpService warpService = NucleusAPI.getWarpService().get();
+
+            List<Warp> warps = warpService.getAllWarps();
+            analysisContainer.addValue(getWithIcon("Warp count", Icon.called("map-marker-alt").of(Color.BLUE)), warps.size());
+
+            if (!warps.isEmpty()) {
+                TableContainer warpsTable = new TableContainer(
+                    getWithIcon("Name", Icon.called("map-marker-alt").of(Family.SOLID)),
+                    getWithIcon("Description", Icon.called("sticky-note").of(Family.REGULAR)),
+                    getWithIcon("Category", Icon.called("list").of(Family.SOLID))
+                );
+
+                for (Warp warp : warps) {
+                    String description = "None";
+                    String category = "None";
+
+                    if (warp.getDescription().isPresent()) {
+                        description = HtmlUtils.swapColorsToSpan(TextSerializers.FORMATTING_CODE.serialize(warp.getDescription().get()));
```

Comment:
Can be simplified with Optional#map
`String description = warp.getDescription().map(desc -> HtmlUtils.swapColorsToSpan(TextSerializers.FORMATTING_CODE.serialize(desc))).orElse("None")`